rust/nft-candy-machine/src/lib.rs

```diff
@@ -447,7 +461,17 @@ pub struct AddConfigLines<'info> {
     #[account(signer)]
     authority: AccountInfo<'info>,
 }
+#[derive(Accounts)]
+pub struct WithdrawFunds<'info> {
+    #[account(
+        mut,
+        has_one = authority
+    )]
+    config: ProgramAccount<'info, Config>,
 
+    #[account(signer, address = config.authority)]
```

Comment:
Use Signer, and make this account(mut)

Reply:
I'm unsure what you mean here

Reply:
![image](https://user-images.githubusercontent.com/22843601/139600918-76eef20a-7cb3-48b1-a59b-267b1ed1b265.png)

Reply:
Hdym 'use signer'?

Reply:
AccountInfo becomes Signer, its a new class

Reply:
Oh for the love of pete, fantastic

Reply:
do we still want this tho?

mut, address = config.authority

or does our has_one in the config do this?